synapse/storage/data_stores/main/room.py

```diff
@@ -335,6 +348,45 @@ def is_room_blocked(self, room_id):
             # Sort alphabetically
             order_by_column = "state.name"
             order_by_asc = True
+        elif RoomSortOrder(order_by) == RoomSortOrder.NAME:
+            order_by_column = "state.name"
+            order_by_asc = True
```

Comment:
Is the goal here to eventually remove the `SIZE` and `ALPHABETICAL` options and just have it be via the field names? Seems odd to have multiple choices that do the same thing.

Reply:
It is a compromise. Remove the old choices (`SIZE`, `ALPHABETICAL`) ist not good for compatibility reason. Do not add the new choices (`NAME`, `JOINED_MEMBERS`) creates a new inconsistent selection.

Reply:
OK, that's what I thought, just wanted to make sure we were on the same page! 👍

Reply:
Should we mark the old one as deprecated in documentation?